hack/build-release.sh

```diff
@@ -42,9 +44,12 @@ fi
 gzip -f "${context}/archive.tar"
 
 # Perform the build and release in Docker.
-cat "${context}/archive.tar.gz" | docker run -e "OS_ONLY_BUILD_PLATFORMS=${OS_ONLY_BUILD_PLATFORMS-}" -i --cidfile="${context}/cid" openshift/origin-release
-docker cp $(cat ${context}/cid):/go/src/github.com/openshift/origin/_output/local/releases "${OS_OUTPUT}"
+docker create -e "OS_ONLY_BUILD_PLATFORMS=${OS_ONLY_BUILD_PLATFORMS-}" --cidfile="${context}/cid" "${release_image}" > /dev/null
+container=$(cat ${context}/cid)
+cat "${context}/archive.tar.gz" | docker cp - $container:$(docker inspect -f '{{ index . "Config" "WorkingDir" }}' $container)
+docker start -a $container
+docker cp "${container}:/go/src/github.com/openshift/origin/_output/local/releases" "${OS_OUTPUT}"
 echo "${OS_GIT_COMMIT}" > "${OS_LOCAL_RELEASEPATH}/.commit"
-docker rm $(cat ${context}/cid)
+docker rm $container > /dev/null
```

Comment:
yeah, stdout prints the container id (which we already have), but we want
errors.

On Tue, Jun 14, 2016 at 12:57 PM, Steve Kuznetsov <notifications@github.com>
wrote:

> In hack/build-release.sh
> <https://github.com/openshift/origin/pull/9278#discussion_r67011110>:
>
> >  echo "${OS_GIT_COMMIT}" > "${OS_LOCAL_RELEASEPATH}/.commit"
> > -docker rm $(cat ${context}/cid)
> > +docker rm $container > /dev/null
>
> "${container}"
>
> ... do we only want stdout to be thrown out?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9278/files/35211f1fb3467971632686c0ed663ee4e87c8635#r67011110>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p0sLaKeIevappwitFMPoH8X8soetks5qLt1-gaJpZM4IzMty>
> .
>